.github/workflows/main.yml

```diff
@@ -313,6 +313,22 @@ jobs:
           sudo make $ch_makej install
           bin/ch-run --version
           $ch_prefix/from-tarball/bin/ch-run --version
+          # Verify uninstall works.
+          sudo make uninstall
+          diff -u - <(find $ch_prefix/from-tarball | sort) <<'EOF'
+          /var/tmp/from-tarball
+          /var/tmp/from-tarball/bin
+          /var/tmp/from-tarball/lib
+          /var/tmp/from-tarball/libexec
+          /var/tmp/from-tarball/share
+          /var/tmp/from-tarball/share/doc
+          /var/tmp/from-tarball/share/man
+          /var/tmp/from-tarball/share/man/man1
+          /var/tmp/from-tarball/share/man/man7
+          EOF
+          # Install again.
+          sudo make $ch_makej install
+          $ch_prefix/from-tarball/bin/ch-run --version
```

Comment:
What about doing this in a new step after the install isn't needed any more? Then it won't need to be reinstalled.